### DISCUSSION TITLE
Trying to install Curseforge Modpack

### DISCUSSION THREAD

**Discussion by SpyTheEngineer** [itzg/docker-minecraft-server#1683] [Q&A]
I'm trying to install the VaultHunters Modpack:
 https://www.curseforge.com/minecraft/modpacks/vault-hunters-official-modpack/files/3853291 
Thats the server files, and no matter what i type, i get an error message saying the server files cant be found, can i get some assistance.

Edit: this is the full command im using on my machine to run the server:
docker run -d -it --name VaultHunters -v /home/spy/VaultHunters:/data -p 25563:25565 -e VERSION=1.16.5 -e TYPE=CURSEFORGE -e CF_SERVER_MOD=VaultHunters-OfficialModpack-1.13.4-Server.zip -e EULA=TRUE -e MEMORY=20G itzg/minecraft-server:java8-jdk

**Comment by itzg** [comment]
You may want to try declaring it as a generic pack, such as this

https://github.com/itzg/docker-minecraft-server/blob/master/examples/docker-compose-generic-pack-atm7.yml

**Reply by SpyTheEngineer** [reply]
How would I go about inputting that example into the ubuntu terminal? Sorry, but I'm very new to this kind of stuff.

**Reply by itzg** [reply]
```
docker run -d -it --name VaultHunters -v /home/spy/VaultHunters:/data -p 25563:25565 -e VERSION=1.16.5 -e TYPE=FORGE -e GENERIC_PACK=VaultHunters-OfficialModpack-1.13.4-Server.zip -e EULA=TRUE -e MEMORY=20G itzg/minecraft-server:java8-jdk
```

**Reply by ImagineZA** [reply]
> ```
> docker run -d -it --name VaultHunters -v /home/spy/VaultHunters:/data -p 25563:25565 -e VERSION=1.16.5 -e TYPE=FORGE -e GENERIC_PACK=VaultHunters-OfficialModpack-1.13.4-Server.zip -e EULA=TRUE -e MEMORY=20G itzg/minecraft-server:java8-jdk
> ```

Hey "G",

So I ran this, using a different modpack, but now I get:
[init] Checking if generic packs are up to date
[init] Generic pack(s) are out of date. Re-applying...
[init] ERROR: unsupported archive type: cannot open `LITV2-1.16.5-1.36.zip' (No such file or directory)

It looks like there is a comma in there that is the incorrect type?

**Reply by itzg** [reply]
@ImagineZA 

> LITV2-1.16.5-1.36.zip' (No such file or directory)

Make sure the file actually exists and is in the correct directory mounted into the container. Also, your `docker run` snippet refers to a totally different modpack, VaultHunters.

**Reply by ImagineZA** [reply]
@itzg 
Here was the docker run I used:
`sudo docker run -d -it -p 25265:25565 -v /etc/minecraft/minecraft_mod:/data -e VERSION=1.16.5 -e TYPE=FORGE -e GENERIC_PACK=/home/user/Minecraft_Stuff/Minecraft_Mods/modpacks/LITV2-1.16.5-1.36.zip -e EULA=TRUE -e MEMORY=8G -m 8G --name name -e EULA=TRUE --restart unless-stopped itzg/minecraft-server`


I tried both using a long location as above and without the location. The LITV2.zip is also in /world/data
